src/master-playlist-controller.js

```diff
@@ -429,8 +402,17 @@ export class MasterPlaylistController extends videojs.EventTarget {
       let updatedPlaylist = this.masterPlaylistLoader_.media();
 
       if (!updatedPlaylist) {
-        // select the initial variant
-        this.initialMedia_ = this.selectPlaylist();
+        let selectedMedia;
```

Comment:
Just a note here.

One of the tests was breaking on this before (https://github.com/videojs/videojs-contrib-hls/blob/fast-start-time/test/videojs-contrib-hls.test.js#L1221) (I didn't have the temp variable, and instead set this.initialMedia_ appropriately).

Once I set things to the temporary variable the problem resolved itself.  I'm assuming there are side effects off of setting this.initialMedia_... I didn't really dig deep enough to understand what was going on.  It seems a little dangerous to have side effects off of setting a variable (when the setter isn't explicitly a function).